packages/flutter/lib/src/material/drop_down.dart

```diff
@@ -411,28 +412,31 @@ class DropdownButtonHideUnderline extends InheritedWidget {
 class DropdownButton<T> extends StatefulWidget {
```

Comment:
at some point we should document `T` here and in the related classes (not necessarily in this PR unless you're feeling particularly heroic)